gatsby/content/docs/jobs-room.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: The Matrix Jobs Room
+slug: matrix-jobs-room
+sort_order: 1000
+section: Introduction
+---
+
+The Matrix Jobs Room ([#jobs:matrix.org](https://matrix.to/#/#jobs:matrix.org)) 
+is a public room where vendors can post jobs and potential customers can apply
```

Comment:
Is this backwards?  Isn't it customers who post jobs, and vendors apply to take them?

Reply:
Yes it is... I'll fix in master